next.config.js

```diff
@@ -1,2 +1,5 @@
+const withPlugins = require('next-compose-plugins')
+const optimizedImages = require('next-optimized-images')
 const withCSS = require('@zeit/next-css')
-module.exports = withCSS()
+// module.exports = withImages(withCSS())
```

Comment:
Let’s remove this comment if we don’t need it :)

Reply:
My bad 😕